src/python/pants/option/global_options.py

```diff
@@ -162,12 +162,7 @@ def register_bootstrap_options(cls, register):
         default_rel_distdir = f"/{default_distdir_name}/"
 
         register(
-            "-l",
-            "--level",
-            type=LogLevel,
-            default=LogLevel.INFO,
-            recursive=True,
```

Comment:
Maybe save this for a followup so that it shows up in the changelog as a distinct change. Stu has been planning to remove recursive options.